RickAndMorty/Models/RMEpisode.swift

```diff
@@ -0,0 +1,49 @@
+//
+//  RMEpisode.swift
+//  RickAndMorty
+//
+//  Created by Juan Camilo Fonseca Gomez on 11/07/23.
+//
+
+import Foundation
+
+struct RMEpisode: Decodable {
+    var id: Int
+    var name: String
+    var air_date: String
+    var episode: String
+    var characterIds: [Int]
+    var url: String
+    var created: Date
+    
+    enum CodingKeys: String, CodingKey {
+        case characters,id,name,air_date,episode, url, created
+    }
+    
+    init(from decoder: Decoder) throws {
+        let container = try decoder.container(keyedBy: CodingKeys.self)
+        let urlFromChar = try container.decode([String].self, forKey: CodingKeys.characters)
+        id = try container.decode(Int.self, forKey: CodingKeys.id)
+        name = try container.decode(String.self, forKey: CodingKeys.name)
+        air_date = try container.decode(String.self, forKey: CodingKeys.air_date)
+        episode = try container.decode(String.self, forKey: CodingKeys.episode)
+        url = try container.decode(String.self, forKey: CodingKeys.url)
+        created = try container.decode(Date.self, forKey: CodingKeys.created)
+        let ids = urlFromChar.map({ (urlGet) in
```

Comment:
You can do this in 1 line using compactMap and $ notation

Reply:
Done